extensions/resteasy-reactive/quarkus-resteasy-reactive-links/deployment/src/test/java/io/quarkus/resteasy/reactive/links/deployment/RestLinksWithFailureInjectionTest.java

```diff
@@ -0,0 +1,31 @@
+package io.quarkus.resteasy.reactive.links.deployment;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+import io.quarkus.test.common.http.TestHTTPResource;
+
+public class RestLinksWithFailureInjectionTest {
+
+    @RegisterExtension
+    static final QuarkusUnitTest TEST = new QuarkusUnitTest()
+            .withApplicationRoot((jar) -> jar
+                    .addClasses(TestRecordNoId.class, TestResourceNoId.class));
```

Comment:
I think what you're looking for in this case is `QuarkusUnitTest#assertException()`.

And in your `@Test` method, just do a `fail()` as it should never go there.

Reply:
Great! That worked